modules/checking-mco-status.adoc

```diff
@@ -14,35 +14,53 @@ To see the status of the Machine Config Operator, its sub-components, and the re
-...
-rendered-master-dde...           2c9371fbb673b97a6fe8b1c52...   3.2.0            5h18m
-rendered-worker-fde...           2c9371fbb673b97a6fe8b1c52...   3.2.0            5h18m
+NAME      CONFIG                    UPDATED  UPDATING   DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT DEGRADEDMACHINECOUNT  AGE
+master    rendered-master-06c9c4…   True     False      False     3             3                  3                   0                     4h42m
+worker    rendered-worker-f4b64…    False    True       False     3             2                  2                   0                     4h42m
 ----
 +
-Note that the `machineconfigs` listed as `rendered` are not meant to be changed or deleted. Expect them to be hidden at some point in the future.
+where:
++
+--
+** `UPDATED`. The `True` status indicates that the MCO has applied the current machine config, as specified in the `STATUS` field in the `oc get mcp` output, to the nodes in that MCP. The `False` status indicates a node in the MCP is updating.
+** `UPDATING`. The `True` status indicates that the MCO is currently applying the desired machine config, as specified in the `MachineConfigPool` custom resource, to at least one of the nodes in that MCP. The desired machine config is the new, edited machine config. Nodes that are updating might not be available for scheduling. The `False` status indicates that all nodes in the MCP are updated. 
+** `DEGRADED`. A `True` status indicates the MCO is blocked from applying the current or desired machine config to at least one of the nodes in that MCP, or the configuration is failing. Nodes that are degraded might not be available for scheduling. A `False` status indicates that all nodes in the MCP are ready. 
+** `MACHINECOUNT`. Indicates the total number of machines in that MCP.
+** `READYMACHINECOUNT`. Indicates the total number of machines in that MCP that are ready for scheduling.
+** `UPDATEDMACHINECOUNT`. Indicates the total number of machines in that MCP that have the current machine config.
+** `DEGRADEDMACHINECOUNT`. Indicates the total number of machines in that MCP that are marked as degraded or unreconcilable.
+--
++
+In the previous output, there are three control plane (master) nodes and three worker nodes. The control plane MCP and the associated nodes are updated to the current machine config. The nodes in the worker MCP are being updated to the desired machine config. Two of the nodes in the worker MCP are updated and one is still updating, as indicated by the `UPDATEDMACHINECOUNT` being `2`. There are no issues, as indicated by the `DEGRADEDMACHINECOUNT` being `0` and `DEGRADED` being `False`. 
++
+While the nodes in MCP are updating, the machine config listed under `CONFIG` is the current machine config, which the MCP is being updated from. When the update is complete, the listed machine config is the desired machine config, which the MCP was updated to.
```

Comment:
You could update "in MCP" to "in the MCP".

Reply:
Resolved